### DISCUSSION TITLE
Tracking page hits and discussion usage

### DISCUSSION THREAD

**Discussion by pete0emerson** [community/community#3501] [Discussions]
It would be great to get page hits and general web usage statistics for our discussions in Github Discussions.

We're looking at using Discussions as a public facing knowledge base. One of the things we'd like to be able to do is track how much our discussions are being viewed, going beyond the number of likes any discussion has.

It looks to me like I could add a pixel beacon (`[![Beacon](URL)](URL)`) to every discussion via a bot to get what I need common web page analytics (Google Analytics et al). Is there anything policy-wise that would prevent me from doing so?

A native solution here would certainly be preferred (hence the feedback), but I'm also up for other creative solutions if anyone has them.

**Comment by Eli112358** [comment]
What about using [Matomo](https://matomo.org)?
